server/db/index.js

```diff
@@ -22,7 +22,7 @@ const jsSchema = require('js-schema')
 
 const HOST = process.env.DB_HOST || 'localhost'
 const PORT = process.env.DB_PORT || 28015
-const NAME = process.env.DB_NAME || 'tnt'
+const NAME = process.env.DB_NAME || 'sc'
```

Comment:
This should be the full word for the db name: `supply_chain`, no reason this should be short - it's not informative enough.